high-school/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,11 @@
+eclipse.preferences.version=1
```

Comment:
I see this message "Sorry, we could not display the entire diff because too many files (376) changed." Something is wrong, please check